src/actions/index.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { projectConfig } from '../config';
+
+const API_KEY = projectConfig.API_KEY;
+const ROOT_URL = projectConfig.ROOT_URL;
+
+export const FETCH_WEATHER = 'FETCH_WEATHER';
+
+export function fetchWeather(city) {
+
+    const url = `${ROOT_URL}?q=${city}&APPID=${API_KEY}`;
```

Comment:
the `url` here is a very generic variable name, can we please make sure to call it something more meaningful.